EIPS/eip-1155.md

```diff
@@ -32,30 +32,30 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 **Smart contracts implementing the ERC-1155 standard MUST implement the `ERC1155` and `ERC165` interfaces.**
 
-```solidity
+```javascript
```

Comment:
Why is this better? Github should have solidity highlighting.

Reply:
Github should have indeed, but it's been 3-4 years that people are asking and there is still no native support for Solidity syntax highlight.

Reply:
I think there is? Even works for files as long as one sets up the `.gitattributes` file
> *.sol linguist-language=Solidity

Reply:
Yes, that would work for .Sol files, but I don't think that works in `.md` files like EIPs

Reply:
I think does.

However it may not be supported by the generator tool used for `eips.ethereum.org.`

Reply:
If you look at the current version of this EIP, you will see that the solidity syntax highlight is not working. Perhaps we need to add the .gitattribute file in root repository, but I'm skeptical that it would work because I personally had many issues in the past with Solidity syntax highlight on GitHub

Reply:
There is [support](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L4746) for Solidity in GitHub Markdown.

But I also see that it doesn't seem to properly highlight it. You might be able to fix that here: 
https://github.com/github/linguist#theres-a-problem-with-the-syntax-highlighting-of-a-file